clientSide/src/components/DropDown/DropDown.css

```diff
@@ -0,0 +1,18 @@
+.drop {
+  border: hidden;
+  box-shadow: 3px 4px 7px var(--global-black);
+  width: 152px;
+  height: 35px;
+  font-family: var(--mont-bee);
```

Comment:
When we use a composite variable like --mont-bee which is a combination of font-family and font-size we need to call the `font: `attribute instead of `font-family: `. Otherwise you can split them using both `font-family` and `font-size` and calling the variables accordingly (: